scripts/npcs/casern_cellar/prisonerasher.lua

```diff
@@ -0,0 +1,99 @@
+--[[
+
+  Prisoner Asher
+  Relationships: Son of Millicent and Borin
+
+  Copyright (C) 2014 Jessica Beller
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--]]
+
+local function prisoner_talk(npc, ch)
+    local function ask_favor()
+        say("Can you please deliver a message to my mother? Her name is Millicent and she lives on the northern end of Goldenfields.")
+        say("Please tell here that I'm ok and not hurt. She must be so worried.")
+        say("And tell her that I'm sorry that I lost control. "
+            .."I made everything worse by trying to throw out those soldiers and getting myself arrested.")
+        say("I really wish I could do something to help her.")
+        local choices = {
+            "Ok, I will tell her.",
+            "What happened to you?"
```

Comment:
Note that you can place `,` behind the second line too, making it a bit easier to extend in the future (not worth the effort to adapt now, but it might make sense to keep in mind for later).